src/components/CharacterCard/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {View} from 'react-native';
+import Icon from '../Icon';
+import RowBetween from '../RowBetween';
+import Separator from '../Separator';
+import Text from '../Text';
+import {Container, CharacterImage, CharacterViewText} from './styles';
+import {CharacterCardProps} from './types';
+
+const CharacterCard = ({onPress, item, ...rest}: CharacterCardProps) => {
+    return (
+        <Container {...rest} onPress={onPress}>
+            <CharacterImage source={{uri: item.image}} />
+            <CharacterViewText>
+                <View>
+                    <Text isBold style={{fontSize: 20}}>
```

Comment:
```suggestion
                    <Text isBold size={20}>
```
you can use the `size` prop to deal with fontSize in your Text component